features/_your_contributions/percent_validation.feature

```diff
@@ -0,0 +1,38 @@
+Feature: Informing about contribution percentages
+  In order to gain a better understanding of how my workplace pension is being calculated,
+  As a user of the WPCC tool,
+  I want to see relevant information about the contributions my employer and I can make.
+
+Background:
+  Given I am on the Your Details step
+  When I fill in my details
```

Comment:
I think this test passes because the salary in this step is 35,000. What happens if the salary is 5876?